fs-index/src/index.rs

```diff
@@ -65,6 +65,17 @@ pub struct Timestamped<Item> {
     pub(crate) last_modified: SystemTime,
 }
 
+impl<Item> Timestamped<Item> {
+    pub fn item(&self) -> &Item {
+        &self.item
+    }
+
+    /// Return the last modified time
+    pub fn last_modified(&self) -> SystemTime {
+        self.last_modified
+    }
+}
```

Comment:
Just curious, why do we need this getters?

Reply:
They’re just there to access `Timestamped` fields without accidentally messing with them. Like, if I do calculations on `last_modified`, I don’t want to change the original value by mistake.